packages/st2mistral/rpm/st2mistral.spec

```diff
@@ -59,6 +60,7 @@ Summary: st2 Mistral workflow service
   %{_bindir}/mistral
   /opt/stackstorm/mistral
   %config(noreplace) %{_sysconfdir}/mistral/*
+  %config(noreplace) %{_sysconfdir}/logrotate.d/mistral
```

Comment:
With this, `/etc/logrotate.d/mistral` file is deleted automatically on mistral remove what is good :+1: 

Checking the specifics on `deb` side.